invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides

```diff
@@ -85,6 +85,10 @@
     padding-top: 5px;
     padding-bottom: 5px;
   }
+
+  .column.searchbar-with-filters{
```

Comment:
this can be removed, already merged in [another pr](https://github.com/inveniosoftware/invenio-app-rdm/pull/1399)